### PR TITLE
Add clarifications on how to run examples.

### PR DESCRIPTION
**Pull request by AlveLarsson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,8 +3,26 @@ A collection of frameworks used as a suite of smoke-tests for tauri
...
+
+To change a exsample to be self contained:
+go to /example/src-tauri/cargo.toml
+and change the dependency 
+```toml
+tauri = { path = "../../../../../tauri", features = [ "edge" ] }
+```
+to
+```toml
+tauri = { version = "0.5.1", features = [ "edge" ] }
```
Comment (nothingismagick) [inline README.md:20]: should be "latest" otherwise we'll be maintaining this readme forever

File: README.md [hunk 2]
```diff
@@ -3,8 +3,26 @@ A collection of frameworks used as a suite of smoke-tests for tauri
 
 These examples are a great way to see how Tauri works with a variety of frameworks.
 
-## error reporting
+## How to run 
+How to run
+ensure to read the readme of the example to check that your first-time setup is done.
+
+The examples assume by default that the main Tauri repo is in the root of your drive ( on windows c:/tauri/)
+
```
Comment (nothingismagick) [inline README.md:11]: Maybe clarify that it does this for the Matrix build because this is the repo for our smoketests.

File: README.md [hunk 3]
```diff
@@ -3,8 +3,26 @@ A collection of frameworks used as a suite of smoke-tests for tauri
 
 These examples are a great way to see how Tauri works with a variety of frameworks.
 
-## error reporting
+## How to run 
+How to run
+ensure to read the readme of the example to check that your first-time setup is done.
```
Comment (nothingismagick) [inline README.md:8]:
```suggestion
Ensure that you have read the readme of the example to check that your first-time setup is done.
```

File: README.md [hunk 4]
```diff
@@ -3,8 +3,26 @@ A collection of frameworks used as a suite of smoke-tests for tauri
 
 These examples are a great way to see how Tauri works with a variety of frameworks.
 
-## error reporting
+## How to run 
+How to run
```
Comment (nothingismagick) [inline README.md:7]:
```suggestion
```

File: README.md [hunk 5]
```diff
@@ -3,8 +3,26 @@ A collection of frameworks used as a suite of smoke-tests for tauri
...
+ensure to read the readme of the example to check that your first-time setup is done.
+
+The examples assume by default that the main Tauri repo is in the root of your drive ( on windows c:/tauri/)
+
+To change a exsample to be self contained:
```
Comment (nothingismagick) [inline README.md:12]:
```suggestion
To change an example to be self-contained:
```

File: README.md [hunk 6]
```diff
@@ -3,8 +3,26 @@ A collection of frameworks used as a suite of smoke-tests for tauri
...
+go to /example/src-tauri/cargo.toml
```
Comment (nothingismagick) [inline README.md:13]:
```suggestion
Go to `/%example_path%/src-tauri/cargo.toml`:
```

File: README.md (hunk 1 above)
Comment (AlveLarsson) [reply README.md]: done

**Pull request merged**